levels/lovelace_tower/events.js

```diff
@@ -46,17 +158,6 @@ if (event.name === "triggerAreaWasEntered" && event.target.key === "objectiveBar
   );
 };
 
-// Remove objective barriers
-if (worldState.obj1Complete === true){
```

Comment:
Removed these because this logic is being handled in some of the code from this PR. That being said, I imagine we could use [property binding](https://twilioquest.github.io/extension-docs/api/custom_objects.html#property-binding) for this sort of thing too? Unless we wanted fancy fading/sliding animations, it'd probably be easier in the long run and reduce the amount of repeat code